app/src/main/java/com/chess/clock/dialog/AdjustTimeDialogFragment.java

```diff
@@ -0,0 +1,94 @@
+package com.chess.clock.dialog;
+
+import android.content.res.ColorStateList;
+import android.content.res.Configuration;
+import android.os.Build;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.chess.clock.R;
+import com.chess.clock.activities.BaseActivity;
+import com.chess.clock.entities.AppTheme;
+import com.chess.clock.entities.ClockTime;
+import com.chess.clock.util.ClockUtils;
+
+public class AdjustTimeDialogFragment extends FullScreenDialogFragment {
+
+    public static final String TAG = "AdjustTimeDialogFragment";
+    private static final String ARG_TIME_KEY = "arg_time_key";
+    private static final String ARG_FIRST_PLAYER_KEY = "arg_first_player_key";
+
+    @Override
+    int layoutRes() {
+        return R.layout.dialog_fragment_adjust_time;
+    }
+
+    @Override
+    public int bgColorRes() {
+        return R.color.black_40;
+    }
+
+    public static AdjustTimeDialogFragment newInstance(Long time, Boolean firstPlayer) {
```

Comment:
Nitpick: If we are not planning on handling nulls in `Boolean` I would just pass a primitive (`boolean)` type.